client/src/containers/HomePage/HomePage.jsx

```diff
@@ -78,6 +84,11 @@ class HomePage extends Component {
     //   });
 
     // });
+    const context = this.context;
+    console.log(context);
+
+    this.state.run = context.run;
+    this.state.stepIndex = context.stepIndex;
```

Comment:
when returning from the covid page, check if the home page joyride should start up again where it left off

Reply:
NOTE TO SELF:
name context.run and context.enabled to context.covidJoyrideRun and context.covidHomepageRun